opensrp-brac-hnpp/src/main/java/org/smartregister/brac/hnpp/utils/HnppJsonFormUtils.java

```diff
@@ -1020,9 +1020,10 @@ public static void addLastAnc(JSONObject jsonForm,String baseEntityId,boolean is
 
             stepOne = jsonForm.getJSONObject(org.smartregister.family.util.JsonFormUtils.STEP1);
             JSONArray jsonArray = stepOne.getJSONArray(org.smartregister.family.util.JsonFormUtils.FIELDS);
-            String prevalue = FamilyLibrary.getInstance().context().allSharedPreferences().getPreference(baseEntityId+"_BRAC_ANC");
+            //String prevalue = FamilyLibrary.getInstance().context().allSharedPreferences().getPreference(baseEntityId+"_BRAC_ANC");
```

Comment:
remove this share preference save logic. may be at formparser class baseEntityId+"_BRAC_ANC"